### PR TITLE
workshop first commit

### PR DESCRIPTION
**Pull request by Dineshgawas64**

### CODE REVIEW

**Review by joyrel-vaz — CHANGES_REQUESTED** [review]

File: workshop/test.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        obj.quit_browser()
+    end
+
+    
+    def test_get_element_by_tag_name_negativetest
+        # url = "file:///C:/Users/dines/git/infuse/Practice-Dinesh/workshop/index.html"
+        url = "file:///app/index.html"
+        obj = Framework.new()
+        obj.opening_site_and_maximizing(url)
+        element = obj.get_element_by_tag_name("inputtt")
```
Comment (joyrel-vaz) [inline workshop/test.rb:103]: Each unit test should test only **one** function. This unit test uses the `get_element_by_tag_name` method.

Comment (joyrel-vaz) [reply workshop/test.rb:103]: You can use the driver methods to get the text

Comment (joyrel-vaz) [reply workshop/test.rb:103]: Change other unit tests similarly

File: workshop/Selenium_Framework_2.rb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+   def initialize
+      Selenium::WebDriver::Chrome.driver_path = "/opt/chromedriver-109.0.5414.74/chromedriver"
+      options = Selenium::WebDriver::Chrome::Options.new
+      options.add_argument("--headless")
+      options.add_argument("--no-sandbox")
+      options.add_argument("--window-size=1920,1080")
+      options.add_argument("--disable-gpu")
+      options.add_argument("--disable-dev-shm-usage")
+      @driver = Selenium::WebDriver.for :chrome, options: options
```
[the remark this answers is not in the document]
Comment (joyrel-vaz) [reply workshop/Selenium_Framework_2.rb:16]: This driver can be passed into the constructor

File: workshop/Selenium_Framework_2.rb [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+     def opening_site_and_maximizing(url)
+        @driver.get(url) 
+        @driver.manage.window.maximize
+        return "site opened and maximized"
+     end
+
+     def wait_for(time)
+         sleep(time)
+     end
```
[the remark this answers is not in the document]
Comment (Dineshgawas64) [reply workshop/Selenium_Framework_2.rb:18]: i added sleep just to make automation slow down ,its  not waiting for any element

**Pull request closed without merging**